integration-tests/features/component_analysis_v2.feature

```diff
@@ -0,0 +1,198 @@
+      Then I should find recommended version <recommended-version> in the component analysis
+      Then I should find one or more vulnerabilities in result with valid attributes
+      
+
+
+     Examples: EPV
+     | ecosystem  | package             | version   | recommended-version | 
+     | npm        | ejs                 | 1.0.0     | 3.1.3               |
+     | npm        | marked              | 0.3.5     | 1.1.0               |
+     | npm        | st                  | 0.2.4     | 2.0.0               |
+     | npm        | npmconf             | 0.0.24    | 2.1.3               |
+     | npm        | moment              | 2.15.1    | 2.25.3              |
+     | npm        | mongoose            | 4.2.4     | 5.9.15              |
+     | maven      | io.vertx:vertx-core | 3.5.3     | 4.0.0-milestone4    |
+     | maven      | io.vertx:vertx-core | 3.5.3.CR1 | 4.0.0-milestone4    |          
+     | maven      | io.vertx:vertx-core | 3.5.2     | 4.0.0-milestone4    |          
+     | maven      | io.vertx:vertx-core | 3.5.2.CR3 | 4.0.0-milestone4    |         
+     | maven      | io.vertx:vertx-core | 3.5.2.CR2 | 4.0.0-milestone4    |          
+     | maven      | io.vertx:vertx-core | 3.5.2.CR1 | 4.0.0-milestone4    |          
+     | pypi       | flask               | 0.12      | 1.1.2               | 
+     | pypi       | flask               | 0.12.1    | 1.1.2               | 
+     | pypi       | flask               | 0.12.2    | 1.1.2               | 
+     | pypi       | numpy               | 1.16.0    | 1.18.4              |
+     | pypi       | numpy               | 1.15.4    | 1.18.4              |
+     | pypi       | numpy               | 1.15.3    | 1.18.4              |
+     | pypi       | numpy               | 1.15.2    | 1.18.4              |
+     | pypi       | numpy               | 1.15.1    | 1.18.4              |
+     | pypi       | requests            | 2.19.1    | 2.23.0              |
+     | pypi       | requests            | 2.19.0    | 2.23.0              |
+     | pypi       | requests            | 2.18.4    | 2.23.0              |
+     | pypi       | requests            | 2.18.3    | 2.23.0              |
+     | pypi       | requests            | 2.18.2    | 2.23.0              |
+     | pypi       | requests            | 2.17.3    | 2.23.0              | 
+
+ 
+  
+  Scenario Outline: Check the component analysis V2 REST API endpoint for components without recommendations
+      Given System is running
+      Given Three scale preview service is running
+      When I wait 5 seconds
+      When I acquire the use_key for 3scale
+      Then I should get the proper user_key
+      When I wait 5 seconds
+      When I start v2 component analyses <ecosystem>/<package>/<version> with user_key
+      Then I should get 200 status code
+        And I should receive a valid JSON response
+      Then I should find no recommendation
+
+
+      Examples: EPV
+     | ecosystem  | package  | version |
```

Comment:
Is this the scenario where all versions of the package have vulnerabilities?

Reply:
this is the scenario for no vulnerabilities and recommendation , the first ever scenario checks for vulnerabilities

Reply:
We have a scenario in which all the versions of a package have vulnerabilities. Then we should not get any recommendation. Do we have a test case for that?

Reply:
Can you add a test for this scenario?

Reply:
Will add it

Reply:
added done